src/chrome/content/rules/GitCDN.xyz.xml

```diff
@@ -0,0 +1,13 @@
+<!--
+	Invalid Certificate:
+		ip.gitcdn.xyz
+-->
+<ruleset name="GitCDN.xyz">
+	<target host="gitcdn.xyz" />
+	<target host="www.gitcdn.xyz" />
+	<target host="cdn.gitcdn.xyz" />
```

Comment:
min.gitcdn.xyz

Reply:
@Bisaloo fixed, and added the same subdomain on the other ruleset gitcdn.link https://github.com/EFForg/https-everywhere/pull/15228/